src/page-multi-facility/MultiFacilityImpactDashboard.tsx

```diff
@@ -96,6 +96,8 @@ const ScenarioPanels: React.FC<ScenarioPanelsProps> = (props) => {
   return <ScenarioPanelsDiv>{panelChildren}</ScenarioPanelsDiv>;
 };
 
+// Boolean to switch off "Rate of Projections" tab if necessary
```

Comment:
I think this comment is in the wrong place?

Reply:
Oops yeah. I had the boolean flag within the file before I found feature flags.